### DISCUSSION TITLE
How do I access nx.dev? Is documentation found elsewhere?

### DISCUSSION THREAD

**Discussion by gabrielkunkel** [nrwl/nx#7004] [General]
I'm afraid this is a stupid question, but anytime I try to go to nx.dev, it just says

_This site can’t be reachednx.dev refused to connect.
Try:

Checking the connection
Checking the proxy and the firewall
ERR_CONNECTION_REFUSED_

I've tried it with and without my VPN on different browsers and different computers. I can immediately go to websites like this one without any problem.

I'm assuming it's just me, because I can't find this question elsewhere and all links point to it. I've clicked on links and I've gone to https://nx.dev/ directly typing in the url.

Am I doing something wrong? Is there something special I have to do to access the site?

**Comment by trumbitta** [comment]
Could it be DNS? Try with Google's DNS servers, or OpenDNS?

**Comment by gabrielkunkel** [comment]
It turns out it IS something weird going on with my MacBook, but not my computer computer or phone. So, for some weird reason, there's got to be some kind of "hosts" file that's blocking it or something like that, which is very weird. It doesn't happen with any other website that I know of.

**Reply by aghwotu** [reply]
@gabrielkunkel Did you ever resolve this? I have the same challenge.

**Reply by phemiai** [reply]
same here, I've the same issue

**Reply by gabrielkunkel** [reply]
I bought a new MacBook. Problem solved. 😬😅
